package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "crispy-api",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:WiseEngineering/crispy-api.git",
+  "author": "Vasyl Zdyrko",
+  "license": "MIT",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node src/index.ts",
```

Comment:
not a blocker for now, but need to think about commands for dev/prod server start 

we could even use something like this for dev
`nodemon --watch 'src/**/*.ts' --ext 'ts' --exec 'yarn start'`